test/synchronicity_test.py

```diff
@@ -160,11 +161,12 @@ async def test_generator_async(synchronizer):
     assert lst == [0, 1, 2]
 
     # Make sure cross-loop calls work
-    s2 = synchronizer
+    s2 = Synchronizer()
     gen2_s2 = s2.create_blocking(gen2).aio
     asyncgen = gen2_s2(gen_s, 3)
     lst = [z async for z in asyncgen]
     assert lst == [0, 1, 2]
+    s2._close_loop()
```

Comment:
With this PR, should `s2.__del__()` and `s2._close_loop` be called automatically or is it still explicitly required at the end of this test?

Reply:
It's good to put it at the end of the test to get repeatability, but yes it would get called by the garbage collector "in a timely manner" after the test completes after this patch :D